pkg/client/download_sbom.go

```diff
@@ -39,7 +39,7 @@ func (c *Client) DownloadSBOM(name string, options DownloadSBOMOptions) error {
 	img, err := c.imageFetcher.Fetch(context.Background(), name, image.FetchOptions{Daemon: options.Daemon, PullPolicy: image.PullNever})
 	if err != nil {
 		if errors.Cause(err) == image.ErrNotFound {
-			c.logger.Warnf("if image is saved on a registry run 'docker pull %s' before downloading the SBoM", name)
+			c.logger.Warnf("if image is saved on a registry run with flag '--remote %s' before downloading the SBoM", name)
```

Comment:
Maybe something like:

"if the image is saved on a registry run with the flag '--remote', for example: ' pack sbom download --remote %s'" ?

Reply:
Yes!! Corrected!! @jjbustamante